content/sources/contributions.adoc

```diff
@@ -126,7 +195,10 @@ In order to help developers, a template is available (https://gist.github.com/li
 
 == Pull Requests best practices
 
-As well as good commits messages are important for code base maintainers, good PRs descriptions are important for reviewers. It helps to understand what the developer has done and why. The PR title must start with the Task issue, then describe briefly what the PR does. For example : _TASK-40308 Allow to update kudos message in activity stream_
+As well as good commits messages are important for code base maintainers, good PRs descriptions are important for reviewers. It helps to understand what the developer has done and why. The PR title must start with the keyword fix or feature, then describe briefly what the PR does, then references the task id and the github issue if needed. For example :
+----
+fix: Allow to update kudos message in activity stream - EXO-40308 - meeds-io/meeds#1234
```

Comment:
`meeds-io/meeds#1234` may not be added in the title in fact to make a link to associated issue, see this example : https://github.com/Meeds-io/gatein-portal/commit/b3db0ce3dece039c1b6cae16a040bcd040555622

( Two way links are added in PR and issue )

Reply:
Do you that, if I put meeds-io/meeds#1234 in the title, it will not be changed to a link ?

If we want the issue reference in the release note, we need to have this reference in the title

Reply:
cc @boubaker

Reply:
Yep, it would be better to add the reference of Github issue link in description and the EXO/MEED task issue in title. @rdenarie